client/scripts/controllers/chain/substrate/account.ts

```diff
@@ -454,7 +455,14 @@ export class SubstrateAccount extends Account<SubstrateCoin> {
       `${this.address} updates nominations`,
     );
   }
-
+  public nominateTrans(nominees: string[]) {
```

Comment:
We've been using the "nameTx" convention for transaction functions, so this should be renamed to "nominateTx" if possible.

Reply:
accout controller have `nominateTx` with `nominees: SubstrateAccount[] `parameter. But I needed a function `nominate` with `nominees: string[]` parameter. Can rename to _nominateTrans_ to __nominateTx_?

Reply:
just change our current `nominateTx` to take strings and then make sure all uses of it do a `accounts.map((a) => a.address)` or something before invoking it